app/controllers/events_controller.rb

```diff
@@ -1,4 +1,7 @@
 class EventsController < ApplicationController
+  # skips authentication only for "index" and "show"
```

Comment:
Maybe phrase like "allow unauthorized users to view the events index/main events page", cause we can probably see that 'index' and 'show' are the things changed. Or add that info on? Or skip it, it's kinda a preference thing. #latenightreviews

Reply:
Well, that's a comment. The line underneath is what is doing the action.
`skip_before_action :authenticate_user!, only: %i[index show]`

In fact, it is exactly what is happening... it is skipping authentication for the `index` and `show` methods. I guess skipping the comment (yeah, I pulled it from a stackoverflow) would be fine.

Reply:
I'm fine with there being a comment; it's sometimes good to describe more narratively what's going on than what's underneath. But I know I'm not the most familiar with Ruby, so maybe calling things out explicitly would help me and others out anyway.